compiler/rustc_span/src/hygiene.rs

```diff
@@ -982,6 +982,10 @@ pub struct ExpnData {
     /// Should debuginfo for the macro be collapsed to the outermost expansion site (in other
     /// words, was the macro definition annotated with `#[collapse_debuginfo]`)?
     pub(crate) collapse_debuginfo: bool,
+    /// Whether this macro expansion came from a compiler implemented macro (`rustc_builtin_macro`)
+    /// If this is true, then we do not display the standard note telling people to use the
+    /// `-Zmacro-backtrace` flag.
```

Comment:
Outdated comment, at this point it's no longer relevant when `hide_backtrace` is set to true, only that it hides the backtrace.